libsnark/gadgetlib1/gadgets/hashes/sha256/sha256_components.tcc

```diff
@@ -31,15 +31,14 @@ const unsigned long SHA256_K[64] = {
     0x5b9cca4f, 0x682e6ff3, 0x748f82ee, 0x78a5636f, 0x84c87814, 0x8cc70208,
     0x90befffa, 0xa4506ceb, 0xbef9a3f7, 0xc67178f2};
 
-const unsigned long SHA256_H[8] = {
-    0x6a09e667,
-    0xbb67ae85,
-    0x3c6ef372,
-    0xa54ff53a,
-    0x510e527f,
-    0x9b05688c,
-    0x1f83d9ab,
-    0x5be0cd19};
+const unsigned long SHA256_H[8] = {0x6a09e667,
+                                   0xbb67ae85,
+                                   0x3c6ef372,
+                                   0xa54ff53a,
+                                   0x510e527f,
+                                   0x9b05688c,
+                                   0x1f83d9ab,
+                                   0x5be0cd19};
```

Comment:
This change is unrelated to this PR. Please be careful when running the formatting script manually and then adding and committing files to make sure that the diffs remain related to the PR goal.

Reply:
Point taken. Hooked the clang formatting script to be executed on file save.